frontend/src/v5/ui/routes/dashboard/projects/projectsList/projectListItem/projectListItem.component.tsx

```diff
@@ -14,25 +14,56 @@
  *  You should have received a copy of the GNU Affero General Public License
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { Link, useRouteMatch } from 'react-router-dom';
+import { Link, useRouteMatch, useParams } from 'react-router-dom';
 import { DashboardListItem } from '@components/dashboard/dashboardList';
 import {
 	DashboardListItemRow,
 	DashboardListItemText,
 } from '@components/dashboard/dashboardList/dashboardListItem/components';
-
+import { useDispatch } from 'react-redux';
 import { discardSlash } from '@/v5/services/routing/routing';
+import { DialogsActions } from '@/v5/store/dialogs/dialogs.redux';
+import { FormattedMessage } from 'react-intl';
+import { IProject } from '@/v5/store/projects/projects.redux';
+import { Button } from '@controls/button';
+import { formatMessage } from '@/v5/services/intl';
+import { ProjectsActionsDispatchers } from '@/v5/services/actionsDispatchers/projectsActions.dispatchers';
+import { DashboardParams } from '@/v5/ui/routes/routes.constants';
+
+type ProjectListItemProps = {
+	project: IProject,
+};
 
-export const ProjectListItem = ({ projectId, name }): JSX.Element => {
+export const ProjectListItem = ({ project }: ProjectListItemProps): JSX.Element => {
 	let { url } = useRouteMatch();
+	const { teamspace } = useParams<DashboardParams>();
 	url = discardSlash(url);
+	const dispatch = useDispatch();
+
+	const onClickDelete = (e) => {
+		e.preventDefault();
+		dispatch(DialogsActions.open('delete', {
```

Comment:
Should we create a DialogsActionsDispatcher?

Reply:
I actually already have done, it's just in another branch that hasn't been merged in yet. (#3272, I think)